### PR TITLE
SD-2682 Fetch news items via Content API by querying for phrases

### PR DESCRIPTION
**Pull request by mugurrus**

### CODE REVIEW

File: server/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Cerberus==0.8.1
 Events==0.2.1
-Eve-Elastic==0.2.16
+Eve-Elastic==0.2.17
```
Comment (sivakuna-aap) [inline server/requirements.txt]: Is `eve-elastic 0.2.17` released? I didn't find it in the github

**Comment by plamut** [comment]
Uhm, where's the actual code?

File: server/requirements.txt (hunk 1 above)
Comment (petrjasek) [inline server/requirements.txt]: it is on [pypi](https://libraries.io/pypi/Eve-Elastic)

**Comment by mugurrus** [comment]
The code is in the eve-elastic datalayer

**Comment by plamut** [comment]
OK, that's good.

Have you checked, though, if the API returns a correct error if more than one phrase is submitted?  Also, how are the queries like `q=term1,"unclosed phrase` treated?

**Comment by ioanpocol** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

File: server/publicapi/items/service.py [hunk 2]
```diff
@@ -93,7 +94,10 @@ def get(self, req, lookup):
         req.args['filter'] = json.dumps(date_filter)
         self._set_fields_filter(req)  # Eve's "projection"
 
-        return super().get(req, lookup)
+        try:
+            return super().get(req, lookup)
+        except InvalidSearchString:
+            raise BadParameterValueError('invalid search text')
```
Comment (plamut) [inline server/publicapi/items/service.py]: Details on _why_ the search text was invalid would be helpful to the user, please check if we can use something meaningful from the `InvalidSearchString`'s error message.

**Comment by mugurrus** [comment]
Queries like queries like q=term1,"unclosed phrase are treated, the client receives 10002 (BadParameterValueError) and a message showing with parameter caused the error.

**Comment by plamut** [comment]
Ok, that's good then.

(there should still be a test for this, though, IMO this has been merged prematurely - no offence, please, meant constructively).

**Pull request merged**